docs/docs/cmd/planner/plan/plan-details-get.md

```diff
@@ -0,0 +1,45 @@
+# planner plan details get
+
+Retrieve the planner details about the specified plan.
+
+## Usage
+
+```sh
+m365 planner plan details get [options]
+```
+
+## Options
+
+`-i,--planId [planId]`
+: ID of the plan. Specify either `planId` or `planTitle` but not both.
+
+`-t,--planTitle [planTitle]`
```

Comment:
Let's put a space between the `,` and `--`

Reply:
done